syncmaster/scheduler/transfer_job_manager.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+import asyncio
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from apscheduler.triggers.cron import CronTrigger
+from kombu.exceptions import KombuError
+
+from syncmaster.backend.services.unit_of_work import UnitOfWork
+from syncmaster.db.models import RunType, Status, Transfer
+from syncmaster.exceptions.run import CannotConnectToTaskQueueError
+from syncmaster.scheduler.transfer_fetcher import get_async_session
```

Comment:
Probably should be imported from .utils.

BTW, could we use functions from `syncmaster.db.factory` instead?